app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext, useEffect } from 'react';
+import { GreetingContext } from '../contexts/GreetingContext';
+
+function Greeting() {
+  const { state, dispatch } = useContext(GreetingContext);
+
+  useEffect(() => {
+    // Fetch the greeting from the Rails API
+    fetch('/random_greeting')
```

Comment:
- [Optional] Our URL doesn't really look like an API. Since we would be building APIs in our future projects making a URL API based it very important eg `api/V1/ /random_greeting` :+1: